lessons/systems-administration/sysadmin-linux-navigation.md

```diff
@@ -1 +1,106 @@
-# 
\ No newline at end of file
+# [System Administration] Linux Navigation
+*Subtitle.*
+
+![SCYCA Logo](.rsrc/git-banner.png)
+
+# Objectives
+- Understand how to navigate the Linux Filesystem
+- objective 2
+- objective 3
+
+# Introduction
+Introduction between 100-150 words.
+
+1. [section 1](#section 1)
+2. [section 2](#section 2)
+3. [section 3](#section 3)
+4. [section 4](#section 4)
+5. [section 5](#section 5)
+
+# Lesson
+## Section 1
+If you want to be able to go somewhere, the first thing you need to now is where you are right now.  With Linux, we can figure this out easily by using the `pwd` command, which stands for "print working directory".  When we run the command, we can see that we are currently in the `/home/hackabit` folder.
+
+![pwd Demo](.rsrc/pwd.png)
+
+But how does this actually work?  Well, `pwd` is actually a program that can read `environment variables` from the operating system.  When you run `pwd` it simply prints out the stored value for the environmental variable named similarly, `PWD`.  We can see all our environment variables by using a different command called `env` that prints out all of them.
+  
+![env Demo](.rsrc/env.png)
+
+Most of these don't matter to us right now, so don't worry if you don't understand most of these.  I've highlighted the relevant part here which shows us that `PWD=/home/hackabit.`  That's where the `pwd` takes information from when we use it to figure what what directory we're currently in. 
+
```

Comment:
recommend removing this section for now to avoid overloading the student with information